.drone.yml

```diff
@@ -335,7 +335,7 @@ steps:
     - pull_request
 
 - name: build
-  image: rancher/dapper:v0.5.0
+  image: rancher/dapper:v0.6.0
```

Comment:
As far as I know, newer versions of dapper still crash on arm. We intentionally have not bumped dapper for arm on other pipelines. Perhaps add a node to this effect, and leave the version as-is?

Reply:
Nice shout - made the changes. PTAL